code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -181,13 +181,19 @@ var/list/gaslist_cache = null
 				//world << "post [temperature], [cached_gases["plasma"][MOLES]], [cached_gases["co2"][MOLES]]
 			*/
 	if(holder)
-		if(cached_gases["freon"] && cached_gases["o2"])
-			if(cached_gases["freon"][MOLES] >= 3)
-				if(return_temperature() < 50)
-					holder.freon_gas_act()
-					cached_gases["freon"][MOLES] -= 3
-					cached_gases["o2"][MOLES] += 3
+		if(cached_gases["freon"])
+			if(cached_gases["freon"][MOLES] >= MOLES_PLASMA_VISIBLE)
+				if(holder.freon_gas_act())
+					cached_gases["freon"][MOLES] -= MOLES_PLASMA_VISIBLE
+					if(temperature >= 3)
+						temperature -= 3

```

Comment:
I can't make it start super cold because then you have >1 canister floods the station